src/events/onStatus.js

```diff
@@ -1,7 +1,12 @@
 // @flow
 import { type Env, type JSONValue } from '../types';
+import Runner from '../Runner';
+import Client from '../Client';
 
-export default async function onStatus(env: Env, body: JSONValue) {
-  let statusEvent = env.ci.processStatusWebhook(body);
-  // ...
+export default function onStatus(
+  client: Client,
+  body: JSONValue,
+  runner: Runner
+) {
+  return client.processStatusWebhook(body);
```

Comment:
This feels a bit useless now too.
It would be good if more logic lived here, but the work needs to be done by the HostAdaptor